lib/fpm/package/deb.rb

```diff
@@ -641,29 +641,39 @@ def write_scripts
   end # def write_scripts
 
   def write_conffiles
-    return unless config_files.any?
-
-    # scan all conf file paths for files and add them
     allconfigs = []
-    config_files.each do |path|
+
+    # expand recursively a given path to be put in allconfigs
+    def add_path(path, allconfigs)
```

Comment:
I'm not sure how I feel about passing the `allconfigs` in for the purposes of modification, hmm... Seems like this would be better moved into a separate class where you do configs.add_path(path) instead.

I'm OK if your code stays as-is, for now, while I ponder this :)